robocode.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
+  <id>org.robocode.Robocode</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>EPL-1.0</project_license>
+  <content_rating type="oars-1.1" />
+  <name>Robocode</name>
+  <summary>Build the best - destroy the rest!</summary>
+  <description>
+    <p>
+      Robocode is a programming game, where the goal is to develop a robot battle tank to battle against other tanks in Java.
+    </p>
+    <p>
+      The robot battles are running in real-time and on-screen.
+    </p>
+    <p>
+      It can serve as a platform for exploring AI and machine learning techniques.
+      Or it can be a competitive, addictive hobby that eats up all your time and CPU cycles.
+    </p>
+  </description>
+  <developer_name>Robocode Community</developer_name>
+  <url type="homepage">https://robocode.sourceforge.io/</url>
+  <url type="help">http://robowiki.net/wiki/Robocode_Basics</url>
+  <launchable type="desktop-id">robocode.desktop</launchable>
```

Comment:
```suggestion
  <launchable type="desktop-id">org.robocode.Robocode.desktop</launchable>
```

Reply:
I thought this would be renamed... :man_shrugging: